### PR TITLE
couple of updates based on beta 4

### PR DESCRIPTION
**Pull request by kalexand-rh**
I installed beta 4 and made a few changes to the installation topic.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@wking, when you have a minute, do you have any objection to these text changes?

File: modules/installation-obtaining-installer.adoc [hunk 1]
```diff
@@ -23,23 +23,24 @@ computer.
...
+. Download the installation program for your operating system and place the file
+in the directory where you will store the installation configuration files.
 +
 [IMPORTANT]
 ====
 The installation program creates several files on the computer that you use to
-install your cluster. While you can delete the installation program after you
-install a cluster, you must keep the files that the installation program
-creates.
+install your cluster. You must keep both the installation program and the files
```
Comment (wking) [inline modules/installation-obtaining-installer.adoc]: Well, you can always re-download the installer.  But yeah, keeping it is probably easier.

File: modules/installation-obtaining-installer.adoc [hunk 2]
```diff
@@ -23,23 +23,24 @@ computer.
...
+that the installation program creates after you finish installing the cluster.
 ====
 
+////
 . Set the installation program to be executable:
```
Comment (wking) [inline modules/installation-obtaining-installer.adoc]: No need to comment this out; just remove it.  The old way is not coming back.

File: modules/installation-obtaining-installer.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/installation-obtaining-installer.adoc]: If we're sure that people can find the right versions to successfully uninstall or that any version of the installer will work, I'm happy to change this.

**Pull request merged**